### PR TITLE
changed to postgres db

### PR DESCRIPTION
**Pull request by sashamussich**
@gil27 maybe i am gonna break everything kkkkkk :dancing_women:

### CODE REVIEW

**Comment by gil27** [comment]
@sashamussich, can you add some content inside seeds.rb file to make our life easier?

tip: [faker gem](https://github.com/stympy/faker) can help with data

File: backend/.env_exaple [hunk 1]
```diff
@@ -0,0 +1,2 @@
+NUTELLA_PASSWORD = dummy
+NUTELLA_USERNAME = dummy
```
Comment (gil27) [inline backend/.env_exaple]: Add the secret keys here too.

They are inside [config/secrets.yml](https://github.com/gil27/nutella/blob/master/backend/config/secrets.yml) file

Comment (sashamussich) [inline backend/.env_exaple]: cool :)))

**Comment by gil27** [comment]
Solves #7

**Pull request merged**